src/v2/Apps/Partner/Routes/Overview/index.tsx

```diff
@@ -15,15 +15,19 @@ const Overview: React.FC<OverviewProps> = ({ partner }) => {
     displayArtistsSection,
   } = partner
 
+  const AreArticlesExisted = !!articles.length
```

Comment:
I'd re-name/case this variable `const hasArticles = articles.length > 0` (or just inline it). The capitalized casing should be reserved for classes/components.

Reply:
whoops, didn't notice it is PascalCased